src/languages/excel.js

```diff
@@ -263,19 +271,18 @@ export default function(hljs) {
     "ISNONTEXT",
     "ISNUMBER",
     "ISODD",
+    "ISOMITTED",
     "ISREF",
     "ISTEXT",
     "ISO.CEILING",
     "ISOWEEKNUM",
     "ISPMT",
     "JIS",
     "KURT",
+    "LAMBDA",
     "LARGE",
     "LCM",
-    "LEFT",
-    "LEFTB",
-    "LEN",
-    "LENB",
```

Comment:
Please double check, these are included in the very link you provided.  MS is well known for forever-level-compatibility... I think if we're going to remove keywords we may need some explicit links/docs showing that microsoft has removed/deprecated certain functions.

Reply:
Apologies, I copied the wrong list over, thanks for picking up.  I'll fix and also make some comments about those being removed, or rlse just leave them all in.

Reply:
Ok, I've done a further commit to this branch which I hope resolves this.